pep-0704.rst

```diff
@@ -0,0 +1,137 @@
+PEP: 704
+Title: Require virtual environments by default for installers
```

Comment:
```suggestion
Title: Require virtual environments by default for package installers
```

@ned-deily suggested this to avoid any confusion with the installers for Python itself, and I agree FWIW